### ISSUE TITLE
Add components to Figma

### ISSUE THREAD

**Issue by staton-hyse11** [grafana/grafana#58720]

**Comment by staton-hyse11** [comment]
@Aires-Grafana , this would be good to look through to validate what components from Storybook are still needed to move over.

**Comment by Aires-Grafana** [comment]
I agree. My suggestion is to make this part of the audit epic #78864 for the next quarter.
